### ISSUE TITLE
eval speed

### ISSUE THREAD

**Issue by kakaluote** [xiaoyufenfei/LEDNet#22]
i run test/eval_forward_time.py  with default params, the mean time is 28ms,  when i set image size (2048,1024) ,the mean time is 100ms.

when i set batchsize to 2,4,8,  the mean time is all 32ms, so the inference time increases linearly

**Comment by xiaoyufenfei** [comment]
Forward time is the time output by eval_forwardTime.py with the model by absorving the batchnorm layers and running with batch_size=1 with the network in eval mode. Training and test sets should be the same, you can select the resolution in the script but normally we use 1024x512.
